fbs/rtpParameters.fbs

```diff
@@ -0,0 +1,91 @@
+
+namespace FBS.RtpParameters;
+
+enum MediaKind : uint8 { ALL = 0, AUDIO, VIDEO }
+enum Type: uint8 { NONE = 0, SIMPLE, SIMULCAST, SVC, PIPE }
+
+// Boolean is a uint8/byte type.
+table Boolean {
+  value:uint8;
+}
+
+table Integer {
+  value:int32;
+}
+
+table IntegerArray {
+  value:[int32];
+}
+
+table Double {
+  value:double;
+}
+
+table String {
+  value:string;
+}
+
+union Value {
+  Boolean,
+  Integer,
+  Double,
+  String,
+  IntegerArray,
+}
+
+table Parameter {
+  name: string;
+  value: Value;
+}
+
+table RtcpFeedback {
+  type:string (required);
+  parameter:string;
+}
+
+table RtpCodecParameters {
+  mime_type:string (required);
```

Comment:
Wondering why underscores instead of camelCase

Reply:
This is a requirement in flatbuffers unless we disable the compile warnings I'm afraid.